homeassistant/components/verisure.py

```diff
@@ -0,0 +1,89 @@
+"""
+components.verisure
+~~~~~~~~~~~~~~~~~~
+"""
+import logging
+from datetime import timedelta
+
+from homeassistant.helpers import validate_config
+from homeassistant.util import Throttle
+from homeassistant.const import (
+    EVENT_HOMEASSISTANT_START, EVENT_HOMEASSISTANT_STOP,
+    CONF_USERNAME, CONF_PASSWORD)
+
+DOMAIN = "verisure"
+DEPENDENCIES = []
+REQUIREMENTS = [
+    'https://github.com/persandstrom/python-verisure/archive/master.zip'
+    ]
+
+_LOGGER = logging.getLogger(__name__)
+
+MY_PAGES = None
+STATUS = {}
+
+MIN_TIME_BETWEEN_REQUESTS = timedelta(seconds=5)
+
+
+def setup(hass, config):
+    """ Setup the Verisure component. """
+
+    if not validate_config(config,
+                           {DOMAIN: [CONF_USERNAME, CONF_PASSWORD]},
+                           _LOGGER):
+        return False
+
+    from verisure import MyPages
```

Comment:
This is not needed anymore. The component will not load if the packages from `REQUIREMENTS` are not installed.